instagram/src/components/Post.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import CommentSection from './CommentSection';
+
+class Post extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { }
+    }
+
+    render() { 
+        const { thumbnail, username, image } = this.props; // same as this.props.thumbnail and so on - 
+        return ( 
+            <div>
+                <img className='user-icon' src={thumbnail}/>
```

Comment:
You need to add likes to the component here. Also, note that images being rendered in JSX requires different syntax than HTML. 

```suggestion
                <img className='user-icon' src={thumbnail}/>
                <h1  className='user-name'>{username}</h1>
                <img className='post-image' src={require({imageUrl})} />
                 <div className="likes">{this.state.likes}</div>
                <CommentSection />
               
```